.k3d-kyma.yaml

```diff
@@ -12,3 +12,15 @@ registries:
     name: kyma
     hostPort: '5001'
 
+options:
+  k3s:
+    nodeLabels:
+      - label: topology.kubernetes.io/region=kyma-local
+        nodeFilters:
+          - server:*
+      - label: topology.kubernetes.io/zone=kyma-local
```

Comment:
The label values are not used anywhere else in the PR? are you using them for testing?

Reply:
The label values not used anywhere, there is a single integration test, testing the existence of the expected attributes based on the labels on different infrastructure/cloud provider.